frontends/pytorch/csrc/c10_dispatch/acap_dispatch.cpp

```diff
@@ -75,7 +79,7 @@ void AcapController::returns(std::vector<at::Tensor> tensors) {
       c10::impl::ExcludeDispatchKeyGuard exclusion(kAcapDispatchKey);
       std::stringstream msg;
       msg << "Cannot return a tensor that is not from the capture context: ";
-      msg << tensor;
+      // msg << tensor;
```

Comment:
remove?

Reply:
Added back (commented out for debugging)